src/Sylius/Bundle/UiBundle/Resources/translations/messages.en.yml

```diff
@@ -26,6 +26,14 @@ sylius:
         change_your_password: Change your password
         charges_and_taxes: Charges and taxes
         checkout: Checkout
+        checkout_addressing_title: Addressing
+        checkout_addressing_description: Specify your billing and shipping address
+        checkout_shipping_title: Shipping
+        checkout_shipping_description: Choose your shipping method
+        checkout_payment_title: Payment
+        checkout_payment_description: Choose your payment method
+        checkout_summary_title: Summary
+        checkout_summary_description: Confirm your order
```

Comment:
Keys should always correspond to their value, we do not have "meta" keys in ``sylius.ui.*`` namespace. I'd say leave them not translated for now, we will tweak general checkout UI after whole process is done.